.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_AUTH0_DOMAIN=dev-ziedxe-0.us.auth0.com
```

Comment:
Nice work using environment variable here. However, you do not want to commit your `.env` file if you can help it (as to keep the keys secret from anyone looking through GitHub).

That being said, the keys will always be visible in your source code through the browser, but you can't prevent that, and that's ok.